src/python/grapl-template-generator/grapl_template_generator/grapl-templates/grapl-python-service-template/cookiecutter.json

```diff
@@ -0,0 +1,6 @@
+{
+    "project_name": "{{ cookiecutter.project_name }}",
+    "project_slug": "{{ cookiecutter.project_slug }}",
+    "pkg_name": "{{ cookiecutter.pkg_name }}",
+    "lambda_handler": "{{ cookiecutter.lambda_handler }}"
+}
```

Comment:
So, I'm completely confused as to _why_ this json file has to exist (i mean, the keys and the values are.... basically the same...) , but stuff screws up if I remove it so 🤷